docs/integrations/global-intelligence/kubernetes-devops.md

```diff
@@ -0,0 +1,180 @@
+---
+
+The Global Intelligence for Kubernetes DevOps application helps infrastructure engineers and DevOps users benchmark the maturity of their Kubernetes adoption and minimize risk of Kubernetes infrastructure incidents by optimizing resource requests and limits for containers.
+
+## Availability
+
+This feature is available in the following account plans.
+
+| Account Type | Account Level                                 |
+|:-------------|:----------------------------------------------|
+| CloudFlex    | Trial, Enterprise                             |
+| Credits      | Trial, Enterprise Operations, Enterprise Suite|
+
+## Data Sources 
+
+Global Intelligence for Kubernetes DevOps App uses metrics data from Kubernetes clusters. 
+
+## Sample Query 
+
+The following sample query is from the **Deployment Count** panel of **GI Kubernetes DevOps - 01. Adoption Stats** dashboard.
+
+```sql
+metric=kube_deployment_spec_replicas _origin=kubernetes | "benchmark-deployment-count" as benchmarkname | quantize 15m using avg | count by deployment, benchmarkname | count by benchmarkname | infer model=benchmark category= k8sdevops //id=deployment_count
+```
+
+## Collecting Metrics for the Global Intelligence for Kubernetes DevOps App
+
+The Sumo Global Intelligence for Kubernetes DevOps app provides insights into your key Kubernetes infrastructure. You can review the metrics collection process and start collecting data.
+
+**Collection Process Overview:** With this graphic, you can see how to collect metrics from your Kubernetes environment and send them to Sumo Logic.
+
+![img](/img/kubernetes/collection-process-overview.png)
+
+Follow the steps on the [Sumo Logic Kubernetes Deployment guide](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/deploy#metrics) to configure the collection for Kubernetes Benchmark metrics.
+
+## Installing the Global Intelligence for Kubernetes DevOps App and Viewing Dashboards
+
+Below are the instructions for installing the Sumo App for Kubernetes DevOps as well as descriptions of each of the app dashboards. 
+
+### Dashboard Filters  
+
+**Each dashboard has a set of filters** that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard. 
+
+:::note
+You can use filters to drill down and examine the data on a granular level. 
+:::
+
+![img](/img/kubernetes/kubernetes-dashboard-filter.png)
+
+Select a **cluster**, **namespace**, and **deployment**. The **history_percent** represents the percent of 15m epocks over the prior 8 days over which Kubernetes metrics data was available for analysis for a given deployment-container combination. Percentages less than 100% represent deployment-container combinations that were new, transient or otherwise inactive for some portion of the prior 8 days.
```

Comment:
```suggestion
Select a **cluster**, **namespace**, and **deployment**. The **history_percent** represents the percent of 15m epochs over the prior 8 days over which Kubernetes metrics data was available for analysis for a given deployment-container combination. Percentages less than 100% represent deployment-container combinations that were new, transient or otherwise inactive for some portion of the prior 8 days.
```